docker/package/scripts/tezos-baker-start

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+. "./tezos.inc.sh"
```

Comment:
also, AFAIU, this won't work cause it will try to use `tezos.inc.sh` from the directory where user did `systemtcl start tezos-bla`